tools/isobuild/package-source.js

```diff
@@ -34,6 +35,34 @@ import {
   optimisticStatOrNull,
 } from "../fs/optimistic.js";
 
+
+// handle .meteorignore
+function handleMeteorIgnore(source) {
+    const dotIgnoreFilePath = files.cwd() + "/.meteorignore";
+    const filesToIgnore =
+        existsSync(dotIgnoreFilePath) && readFileSync(dotIgnoreFilePath).toString();
+
+    if (filesToIgnore === false) return source;
+
+    const filesInDotMeteorIgnore = filesToIgnore
+        .split(/\n/)
+        .map(line => {
+            try {
+              const _line = line
+                  .replace(/[\-\[\]\/\{\}\(\)\*\+\?\.\\\^\$\|]/g, "\\$&")
```

Comment:
Without getting into the rest of the PR right now, I'll just point out that most of these characters in this character set (`[...]`) do not need to be escaped (because they are inside the set).  In other words:

```js
/[\][\\/{}()*+?^$|.-]/
```

(with just `]` and `\` needing to be escaped, and a couple changes to the order)

Reply:
Hmm, thanks. I'm editing this file anyway :)